index.bs

```diff
@@ -268,33 +279,40 @@ xrDevice.requestSession({ immersive: true }).then((session) => {
 Session {#session}
 =======
 
-XRSessionCreationOptions {#xrsessioncreationoptions-interface}
+XRSessionMode {#xrsessionmode-enum}
 -------------------------
 
 <pre class="idl">
-dictionary XRSessionCreationOptions {
-  boolean immersive = false;
-  XRPresentationContext outputContext;
+enum XRSessionMode {
+  "inline",
+  "immersive-vr",
+  "immersive-ar"
 };
 </pre>
 
-The {{XRSessionCreationOptions}} dictionary provides a <dfn>session description</dfn>, indicating the desired properties of a session to be returned from {{requestSession()}}.
+The {{XRSessionMode}} enum defines the modes that an {{XRSession}} can operate in.
 
-<div class="algorithm" data-algorithm="supported-by-device">
+  - A session mode of <dfn enum-value for="XRSessionMode">inline</dfn> indicates that the session's output will be shown as an element in the HTML document. {{inline}} session content MAY be displayed in mono or stereo and MAY allow for [=viewer=] tracking. User agents MUST allow {{inline}} sessions to be created for any [=XR device=].
+  - A session mode of <dfn enum-value for="XRSessionMode">immersive-vr</dfn> indicates that the session's output will be shown on the [=XR device=] hardware in a manner that occupies a significant portion of the user's vision. The {{environmentBlendMode}} MAY be {{opaque}} for {{immersive-vr}} sessions.
+  -  A session mode of <dfn enum-value for="XRSessionMode">immersive-ar</dfn> indicates that the session's output will be shown either on the [=XR device=] hardware in a manner that occupies a significant portion of the user's vision or fullscreen on hardware that does not occupy a significant portion of the user's vision. The {{environmentBlendMode}} MUST NOT be {{opaque}} for {{immersive-ar}} sessions.
```

Comment:
oof.  I had a hard time understanding that first sentence... Instead, can we talk about it having exclusive access to the xr device display?

Reply:
Yeah, sorry about that one. Taking another crack at it. 🙄